src/leveled_pmem.erl

```diff
@@ -71,7 +71,7 @@ prepare_for_index(IndexArray, no_lookup) ->
 prepare_for_index(IndexArray, Hash) ->
     {Slot, H0} = split_hash(Hash),
     Bin = array:get(Slot, IndexArray),
-    array:set(Slot, <<Bin/binary, 1:1/integer, H0:23/integer>>, IndexArray).
+    array:set(Slot, <<Bin/binary, H0:24/integer>>, IndexArray).
```

Comment:
This change suggests that split_hash always returns a `H0` in which the most significant bit is set. I believe you but hard to digest from the code.

Reply:
It is just that previously we were checking for the leading bit of 1 (in find_pos/2), but it had no meaning in this function (i.e. there was no case for the leading bit of 0).

So we were needlessly matching on just 23 bits of the hash, when we could be doing a simpler match on 24 bits.  The hash can have a leading bit of 0, as find_pos/2 doesn't care.